testapi.pm

```diff
@@ -921,6 +923,11 @@ sub script_output {
     # 80 is approximate quantity of chars typed during 'curl' approach
     # if script length is lower there is no point to proceed with more complex solution
     $args{type_command} ||= length($current_test_script) < 80;
+    # Surround the actual script output with special markers so that we can
+    # unambiguously separate the expected output from other content that we
+    # might encounter on the serial device depending on how it is used in the
+    # SUT
+    $current_test_script = "echo '>>>'$($current_test_script)'<<<'";
```

Comment:
this only works for one lined output. try with script_output('mount') in full-stack.

But we already guard the end of the script with SCRIPT_FINISHED. So if you prepend an echo command to the test script and strip that too - it should work.

Reply:
Well, I'm trying to run `script_output` in the full stack test but TinyCore does not have bash which `script_output` currently relies on … Let's see what I can come up with

Reply:
One more argument to have script_output actually not in os-autoinst but in a linux test library :(

Reply:
anyway, then test manually - all I'm saying it's buggy :)